docker/Dockerfile

```diff
@@ -86,11 +86,15 @@ RUN --mount=type=cache,target=/root/.cache/ccache,id=ccache \
     --mount=type=cache,target=/root/.cache/uv,id=uv-cache,sharing=locked \
     . /opt/venv/bin/activate && \
     if [ "$VLLM_VERSION" = "nightly" ]; then \
-        uv pip install --prerelease=allow 'vllm[runai,tensorizer,flashinfer]' \
+        uv pip install --prerelease=allow \
```

Comment:
(Non-blocking) If VLLM or any dependencies add other url dependency in the future, we would need to make the same change. Do we consider removing uv until uv supports url dependency support?